anise-cli/src/args.rs

```diff
@@ -66,3 +55,17 @@ pub enum Actions {
         id: i32,
     },
 }
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, ArgsT)]
+pub(crate) struct TruncateById {
```

Comment:
Could you add an example of how the command changes now that `TruncateById` derives clap's `Args`?